### ISSUE TITLE
tensorflow.keras.optimizers.Adadelta() Inconsistencies With keras.optimizers.Adadelta()

### ISSUE THREAD

**Issue by ilhamfp** [tensorflow/tensorflow#31024]
**System information**
- OS Platform and Distribution: Linux-4.9.0-8-amd64-x86_64-with-debian-9.9 . 
- TensorFlow version: v1.14.0-9-gc407b045b8 1.14.0 . 
- Python version: 3.6.6 . 

**Describe the current behavior**  
Take a look at the following two Kaggle kernel:  
1. https://www.kaggle.com/ilhamfp31/keras-mnist  
2. https://www.kaggle.com/ilhamfp31/tensorflow-keras-mnist

Both of them run the same code with the same OS, python version and TensorFlow version. The only difference is the first one importing from keras and the second one from tensorflow.keras. Optimizer Adadelta produces a different result because of the default configuration is not the same. The difference lies in the learning rate.

`>> print(keras.optimizers.Adadelta().get_config())`
`{'lr': 1.0, 'rho': 0.95, 'decay': 0.0, 'epsilon': 1e-07}`

`>> print(tensorflow.keras.optimizers.Adadelta().get_config())`
`{'name': 'Adadelta', 'learning_rate': 0.001, 'decay': 0.0, 'rho': 0.95, 'epsilon': 1e-07}`

**Describe the expected behavior**
`keras.optimizers.Adadelta()` and `tensorflow.keras.optimizers.Adadelta()` should produce the same result as shown by optimizer Adam. Looking at the source code, both Adam optimizer code in [optimizer.py](https://github.com/tensorflow/tensorflow/blob/r1.14/tensorflow/python/keras/optimizers.py) and [optimizer_v2/adam.py](https://github.com/tensorflow/tensorflow/blob/r1.14/tensorflow/python/keras/optimizer_v2/adam.py) default parameter is consistent with [keras optimizer code](https://github.com/keras-team/keras/blob/master/keras/optimizers.py). This is not true with Adadelta. While the code in [optimizer.py](https://github.com/tensorflow/tensorflow/blob/r1.14/tensorflow/python/keras/optimizers.py) is consistent, the code in [optimizer_v2/adadelta.py](https://github.com/tensorflow/tensorflow/blob/r1.14/tensorflow/python/keras/optimizer_v2/adadelta.py) is not. 

**Code to reproduce the issue**
Code needed to reproduce the issue is available by downloading it from the given Kaggle kernel link. Simply click three grey dot on the upper right corner and click `Download code`.

**Comment by ilhamfp** [comment]
I have create a [pull request](https://github.com/tensorflow/tensorflow/pull/31025) fixing this issue.

**Comment by ymodak** [comment]
Discussion on the [PR thread](https://github.com/tensorflow/tensorflow/pull/31025) addresses this issue. Hence closing it. Feel free to reopen if necessary. Thanks!
